tests/test_octodns_processor_filter.py

```diff
@@ -161,6 +163,38 @@ def test_regex(self):
         )
 
 
+class TestNetworkValueFilter(TestCase):
+    zone = Zone('unit.tests.', [])
+    matches = Record.new(
+        zone, 'private-ipv4', {'type': 'A', 'ttl': 42, 'value': '10.42.42.42'}
+    )
+    zone.add_record(matches)
+    doesnt = Record.new(
+        zone, 'public-ipv4', {'type': 'A', 'ttl': 42, 'value': '42.42.42.42'}
+    )
+    zone.add_record(doesnt)
+    matchable1 = Record.new(
+        zone, 'private-ipv6', {'type': 'AAAA', 'ttl': 42, 'value': 'fd12:3456:789a:1::1'}
+    )
+    zone.add_record(matchable1)
+    matchable2 = Record.new(
+        zone, 'public-ipv6', {'type': 'AAAA', 'ttl': 42, 'value': 'dead:beef:cafe::1'}
+    )
+    zone.add_record(matchable2)
```

Comment:
Probablby best to add a non A/AAAA record here to test things with and verify that it's left alone and remains the the zone. Not really testing the new provider here, but more making sure that the Accepts/RejectsMixin behave as expected in case they get modified at some point in the future.

Reply:
you're absolutely right, will do

Reply:
I've added a simple TXT record to handle that case